routing/routes.go

```diff
@@ -25,27 +25,85 @@ func NewRoutes() (r *Routes) {
 	r = new(Routes)
 
 	r.Routes = []Route{
+		// swagger:route GET / Root
+		// ---
+		// Handler to get the status of Kiali
+		//
+		//     Consumes:
+		//     - application/json
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http
+		// responses:
+		//      200: statusInfo
 		{
 			"Root",
 			"GET",
 			"/api",
 			handlers.Root,
 			false,
 		},
+		// swagger:route GET /token GetToken
+		// ---
+		// Handler to get the authentication token
+		//
+		//     Consumes:
+		//     - application/json
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http
+		//
+		//	   Security:
+		//     authorization: user, password
+		//
+		// responses:
+		//      200: tokenGenerated
 		{ // Request the token
 			"Status",
 			"GET",
 			"/api/token",
 			handlers.GetToken,
 			true,
 		},
-		{ // another way to get to root, both show status
+		// swagger:route GET /status getStatus
+		// ---
+		// Handler to get the status of Kiali
+		//
+		//     Consumes:
+		//     - application/json
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http
+		//
+		// responses:
+		//      200: statusInfo
+		{
 			"Status",
 			"GET",
 			"/api/status",
 			handlers.Root,
 			false,
 		},
+		// swagger:route GET /namespaces/{namespace}/istio istioConfigList
+		// ---
+		// Handler to get the list of Istio Config of a namespace
+		//
+		//     Consumes:
+		//     - application/json
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http
+		//
+		//     responses:
+		//       200: istioConfigList
```

Comment:
Also, good to add the generic error responses.
On this case a 404 is possible and in generic, a 500 should be also a valid error in all endpoints.
For example, in Hawkular we covered all
http://www.hawkular.org/docs/rest/rest-alerts-v2.html#__alerts_handling

Reply:
This comment may apply to the other endpoints as well.

Reply:
OK I'll work on the genericResponse, I'll add a new struct for this like "Response", No refactor need it